examples/window-app/common/window-app.zap

```diff
@@ -3448,6 +3510,36 @@
               "maxInterval": 65344,
               "reportableChange": 0
             }
+          ],
+          "events": [
+            {
+              "name": "HardwareFaultChange",
+              "code": 0,
+              "mfgCode": null,
+              "side": "server",
+              "included": 1
+            },
+            {
+              "name": "RadioFaultChange",
+              "code": 1,
+              "mfgCode": null,
+              "side": "server",
+              "included": 1
+            },
+            {
+              "name": "NetworkFaultChange",
```

Comment:
This is wrong.  This app never emits this event.  Please check these things carefully....  I expect it never emits some of the other events here either.

Reply:
Addressed in #24980